src/utils.js

```diff
@@ -93,6 +93,25 @@ const getSemesterName = (term) => {
   return `${semester} ${year}`;
 };
 
+const getSemesterMonth = (term) => {
```

Comment:
Why was this function added? I can't seem to find any uses of it.

Reply:
When I was renaming `innerValue`, `innerLabel` in `<Select />` to better differentiate option label and identifier, at one point I tried to call onClick for term select using label instead of identifier; I needed to convert label ("Summer 2020" format)  to identifier ("202005" format). I created this helper function. Though in the end I did not need to use it, I left it in the utils.js file, just in case if we need the conversion later.